docs/content/operations/recommendations.md

```diff
@@ -5,6 +5,39 @@ layout: doc_page
 Recommendations
 ===============
 
+# Some Genearl guidelines
+
+JVM Flags..
+```
+-Duser.timezone=UTC
+-Dfile.encoding=UTF-8
+-XX:+PrintGCDetails
+-XX:+PrintGCDateStamps
+-XX:+PrintGCTimeStamps
+-XX:+PrintGCApplicationStoppedTime
+-XX:+PrintGCApplicationConcurrentTime
+-Xloggc:/var/logs/druid/historical.gc.log
+-XX:+UseGCLogFileRotation
+-XX:NumberOfGCLogFiles=50
+-XX:GCLogFileSize=10m
+-XX:+ExitOnOutOfMemoryError
+-XX:+HeapDumpOnOutOfMemoryError
+-XX:HeapDumpPath=/var/logs/druid/historical.hprof
+-XX:MaxDirectMemorySize=10240g
+```
+
+`ExitOnOutOfMemoryError` flag is only supported starting JDK 8u92 . For older versions, `-XX:OnOutOfMemoryError='kill -9 %p'` can be used.
+
+`MaxDirectMemorySize` restricts jvm from allocating more than specified limit, by setting it to unlimited jvm restriction is lifted and OS level memory limits would take effect.
+
+Please note that above flags are general guidelines only, Be cautious and feel free to change them if necessary for the specific deployment.
```

Comment:
"only, Be" isn't grammatical. Suggest "only. Be"

Reply:
fixed